<launch>
    
    <!-- Configuration -->
    <rosparam command="load" file="$(find uw_slam)/config/configuration.yml"/>

    <!-- Main uw_slam -->
    <node pkg="uw_slam" type="uw_slam_node" name="uw_slam_node" output="screen"/>
        
    <!-- Rviz visualizer-->
    <!-- <node pkg="rviz" type="rviz" name="rviz"/> -->

</launch>